lib/src/10_auth/auth_screen.dart

```diff
@@ -333,27 +349,19 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
 
     return Column(
       children: <Widget>[
-        TextField(
-          autofocus: true,
-          controller: emailController,
-          decoration: InputDecoration(
-            labelText: resolvePhrase(FauiPhrases.EmailTextField, "Email"),
-          ),
-          onSubmitted: (s) {
-            submit();
-          },
-        ),
+        _buildTextBox(emailController, _emailNode, null,
+            resolvePhrase(FauiPhrases.EmailTextField, 'Email'), submit),
         _buildError(context, _error),
-        if (_loading == true)
-          AuthProgress(resolvePhrase(
-              FauiPhrases.SendingPasswordResetLinkMessage,
-              'sending password reset link...')),
-        if (_loading == false)
-          RaisedButton(
-            child: Text(resolvePhrase(FauiPhrases.SendPasswordResetLinkButton,
-                'Send Password Reset Link')),
-            onPressed: submit,
-          ),
+        (_loading == true)
+            ? AuthProgress(resolvePhrase(
+                FauiPhrases.SendingPasswordResetLinkMessage,
+                'sending password reset link...'))
+            : RaisedButton(
+                child: Text(resolvePhrase(
+                    FauiPhrases.SendPasswordResetLinkButton,
+                    'Send Password Reset Link')),
```

Comment:
What's the goal of this change?

Reply:
I think I just substituted the previous if else condition by ternary operator.

Reply:
done